src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Tags/DhGaugeParameters.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DhGaugeParameters.hpp"
+#include "Options/Options.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+namespace OptionTags {
+template <bool UseRollon>
+struct DhGaugeParameters {
+  using type = GeneralizedHarmonic::gauges::DhGaugeParameters<UseRollon>;
+  static constexpr Options::String help{
+      "Parameters for initializing damped harmonic gauge."};
+  using group = evolution::OptionTags::SystemGroup;
```

Comment:
Why not `using group = GeneralizedHarmonic::OptionTags::Group;` (also see comment on `KerrSchild.yaml`)?

Reply:
`using group = GeneralizedHarmonic::OptionTags::Group;` is for consistency with `src/Evolution/Systems/GrMhd/ValenciaDivClean/Tags.hpp`

Reply:
If @prayush and @nilsdeppe are both ok with it, I'd like to leave this as-is